lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -228,8 +145,8 @@ def render_to_fragment(self, request, program_uuid, **kwargs):  # lint-amnesty,
             'certificate_data': certificate_data,
             'industry_pathways': industry_pathways,
             'credit_pathways': credit_pathways,
-            'program_discussions_enabled': program_discussions_is_enabled(),
-            'discussion_fragment': self.render_discussions_fragment(program_uuid, request)
+            'program_discussions_enabled': program_discussion_lti.enabled,
+            'discussion_fragment': program_discussion_lti.render_discussions_iframe()
```

Comment:
@asadazam93 has a good point that the template expects the data as a dict, so if you are keeping the data as above you'd need to change templates as well. 

```
...
'discussion' : {
  'iframe': program_discussion_lti.render_discussions_iframe(),
  'enabled': program_discussion_lti.enabled
}
```

given that, there is a need of refactor templates from using `program_discussions_enabled` to `discussion.enabled` flag.